boot/assets_test.go

```diff
@@ -323,13 +323,10 @@ func (s *assetsSuite) TestInstallObserverObserveSystemBootRealGrub(c *C) {
 func (s *assetsSuite) TestInstallObserverObserveSystemBootMocked(c *C) {
 	d := c.MkDir()
 
-	tab := bootloadertest.Mock("trusted-assets", "").WithTrustedAssets()
-	bootloader.Force(tab)
-	defer bootloader.Force(nil)
-	tab.TrustedAssetsList = []string{
+	tab := s.bootloaderWithTrustedAssets(c, []string{
```

Comment:
Do you need to do the equivalent of `defer bootloader.Force(nil)`?

Reply:
No, bootloaderWithTrustedAssets does a relevant s.AddCleanup() call.